.github/workflows/ci.yaml

```diff
@@ -5,3 +5,5 @@ on: ["push", "pull_request"]
 jobs:
   ci:
     uses: ./.github/workflows/global-ci.yml
+    with:
+      ui_tests_ref: 32c44ba1f457f0e87204787b4c81bd6cf1d211fb
```

Comment:
I might miss something, but isn't it better use branch names instead of commit tags to allow test fixes/updates without need to change ```ui_tests_ref``` in this CI repo?

Reply:
for sure, this was just to make sure it picked up the most recent PR. I didn't sign this commit to ensure that this PR won't merge whie it's in it